src/components/web_lighthouse_gauge/web_lighthouse_gauge.ts

```diff
@@ -0,0 +1,50 @@
+// Import the LitElement base class and html helper function
+import {html, property, customElement, PropertyValues} from 'lit-element';
+import {BaseElement} from '../base_element/base_element';
+import './web_lighthouse_gauge.scss';
+
+/**
+ * Lighthouse gauge element.
+ */
+@customElement('web-lighthouse-gauge')
+export class WebProgressBar extends BaseElement {
+  @property({type: Number}) score = 1;
+
+  // eslint-disable-next-line require-jsdoc
+  render() {
+    const score = this.score;
+
+    this.setAttribute('role', 'progressbar');
```

Comment:
Do you know if LitElement has a lifecycle callback like connectedCallback where these could be set? Or is it recommended to do that work in render? Just to avoid calling setAttribute more than once.

Reply:
I think we can probably put these into `firstUpdated` below.